frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,94 @@
-import { Button } from '~/libs/components/components.js';
+import helloImg from '~/assets/img/hello.png';
+import hidePasswordIcon from '~/assets/img/hidePassword.svg';
+import showPasswordIcon from '~/assets/img/showPassword.svg';
+import { Button, Input, Link, Message } from '~/libs/components/components.js';
+import { useAppForm, useCallback, useState } from '~/libs/hooks/hooks.js';
+import {
+  type UserAuthRequestDto,
+  userSignInValidationSchema,
+} from '~/packages/users/users.js';
+import { DEFAULT_LOGIN_PAYLOAD } from '~/pages/auth/components/sign-in-form/libs/common/constants.js';
+import styles from '~/pages/auth/components/sign-in-form/styles.module.scss';
 
 type Properties = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserAuthRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => (
-  <>
-    <h1>Sign In</h1>
-    <form>
-      <Button label="Sign in" />
-    </form>
-  </>
-);
+const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
+  const { control, errors, handleSubmit } = useAppForm({
+    defaultValues: DEFAULT_LOGIN_PAYLOAD,
+    validationSchema: userSignInValidationSchema
+  });
+  const [isPasswordVisible, setPasswordVisibility] = useState(false);
+
+  const togglePasswordVisibility = useCallback((): void => {
+    setPasswordVisibility(!isPasswordVisible);
+  }, [isPasswordVisible]);
+
+  const handleFormSubmit = useCallback(
+    (event_: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event_);
+    },
+    [handleSubmit, onSubmit],
+  );
+
+  return (
+    <>
+      <div className={styles.wrapper}>
+        <img
+          className={styles.helloImg}
+          src={helloImg}
+          alt='hello' />
+        <form
+          className={styles.form}
+          name="loginForm"
+          onSubmit={handleFormSubmit}
+        >
+          <fieldset
```

Comment:
Do you have prettier vscode extension installed? Something is wrong with formatting 🤔